openapi3filter/validate_request.go

```diff
@@ -283,12 +291,15 @@ func validateSecurityRequirement(c context.Context, input *RequestValidationInpu
 			}
 		}
 		scopes := securityRequirement[name]
-		return f(c, &AuthenticationInput{
+		err := f(c, &AuthenticationInput{
 			RequestValidationInput: input,
 			SecuritySchemeName:     name,
 			SecurityScheme:         securityScheme,
 			Scopes:                 scopes,
 		})
+		if err != nil {
```

Comment:
Looks like you can inline this check with the line above.

Reply:
I've done it in d9d4d95